test/runtime/runtime-kotlin/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Gradle-specific files
```

Comment:
I would add the official git ignore for gradle, and of course keep the kotlin specific ignore of build, as well as the src files.

https://github.com/github/gitignore/blob/main/Gradle.gitignore

Remember to remove all the files it now ignores and commit it, otherwise the files will keep being in the PR.

Reply:
it's a logging library for node i have removed it as it is not required